src/main/java/frc/robot/turret/commands/TurretCommands.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.turret.commands;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import frc.robot.RobotContainer;
+import frc.robot.turret.TurretState;
+import frc.robot.turret.TurretSubsystem;
+import frc.utils.joysticks.SmartJoystick;
+
+public abstract class TurretCommands {
```

Comment:
maybe move it outside command package and them you can make the control funcs in turret protected instead of public

Reply:
its to big for this PR i'll might open a new PR for that later